pypdf/_writer.py

```diff
@@ -1349,6 +1371,97 @@ def add_metadata(self, infos: Dict[str, Any]) -> None:
         assert isinstance(self._info, DictionaryObject)
         self._info.update(args)
 
+    def compress_identical_objects(
+        self,
+        remove_identicals: bool = True,
+        remove_orphans: bool = True,
+        verbose: int = -1,
```

Comment:
Do we really need the verbosity parameter?

Reply:
for big files this may help to see progress